packages/js/data/src/product-attribute-terms/index.ts

```diff
@@ -12,3 +20,14 @@ createCrudDataStore( {
 } );
 
 export const EXPERIMENTAL_PRODUCT_ATTRIBUTE_TERMS_STORE_NAME = STORE_NAME;
+
+declare module '@wordpress/data' {
+	// TODO: convert action.js to TS
+	function dispatch( key: typeof STORE_NAME ): ActionDispatchers;
+	function select(
+		key: typeof STORE_NAME
+	): SelectFromMap< ProductAttributeTermsSelectors > & WPDataSelectors;
+	function resolveSelect(
+		key: typeof STORE_NAME
+	): PromiseifySelectors< SelectFromMap< ProductAttributeTermsSelectors > >;
```

Comment:
This helps expose the store action/selector types automatically when using `useSelect` or `useDispatch`